history/spec.md

```diff
@@ -133,19 +132,19 @@ WeakCell.prototype.clear() performs the following steps:
     1. Remove _O_ from _factory_.[[ActiveCells]].
     1. Set _O_.[[Target]] to undefined.
     1. Set _O_.[[Factory]] to undefined.
-    1. Set _O_.[[Holdings]] to undefined.
+    1. Set _O_.[[heldValue]] to undefined.
 1. Return **undefined**.
 
-### get WeakCell.prototype.holdings
+### get WeakCell.prototype.heldValue
```

Comment:
This is a normative change

Reply:
oops, good call

Reply:
Sounds more like a too eager replace all to me. The history document probably shouldn't be changed.

Reply:
It was manual, but i certainly grepped for every instance. I can revert the history changes if that makes sense.

Reply:
Yeah I guess this isn't normative after all because that's renaming the *historical* API, which should be frozen in time. The change to the current API is in fact editorial?

Reply:
Oh right, I actually missed that this was in the historical spec.

Reply:
I've reverted the `history/` changes.